### PR TITLE
Added more resources

### PR DESCRIPTION
**Pull request by nihal-ramaswamy**
Added resources to Ethical Hacking and Standard Template Library

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -312,6 +319,10 @@ To contribute, please check out our contribution guide [here](https://github.com
 
 ## Others
 
+### Standard Template Library
```
Comment (shahednasser) [inline README.md:322]: Add Standard Template Library in the correct alphabetical position under Others.

**Review by shahednasser — CHANGES_REQUESTED** [review]
Please add both items in the content menu

**Comment by nihal-ramaswamy** [comment]
Made the changes

**Comment by shahednasser** [comment]
Great thank you!

**Pull request merged**